tox.ini

```diff
@@ -4,8 +4,8 @@ env_list = lint,smoke
 
 [testenv]
 setenv =
-	version=14.0
-	channel=14.0/edge
+	version=16.6
+	channel=16.6/edge
```

Comment:
16.8 in snapcraft.yaml

Reply:
That shouldn't be used anywhere. Removing.